views/emails/html/update-project.php

```diff
@@ -75,22 +73,18 @@
                                     </div>
                                     <div>
                                         <?php
-                                            printf( 
-                                                '%s "%s" %s %s', 
-                                                __( 'The project', 'wedevs-project-manager'  ),
-                                                $title,
-                                                __( 'has been updated by', 'wedevs-project-manager' ),
-                                                $updater['data']['display_name']
+                                            printf(
+                                                '%s "%s" %s %s',
+                                                esc_html_e( 'The project', 'wedevs-project-manager'  ),
+                                                esc_html( $title ),
+                                                esc_html_e( 'has been updated by', 'wedevs-project-manager' ),
```

Comment:
`_e` is not needed inside `printf`. use `__` instead.
However the sentece should be intact to make a proper and flexible translation.
```
        printf(
            esc_html__( 'The project "%1$s" has been updated by %2$s', 'wedevs-project-manager'  ),
            $title,
            $updater['data']['display_name']
        );
```